### ISSUE TITLE
Release 15.6.1

### ISSUE THREAD

**Issue by mattxwang** [stylelint/stylelint#6829]
- [x] stylelint release
- (n/a) stylelint-config-recommended update/release
- (n/a) stylelint-config-standard update/release
- [x] stylelint.io update
- [x] stylelint-demo check
- [x] tweet

See #6820 for the changelog; we have 3 minor bugfixes coming up. Was thinking that this would be a good opportunity for a release since it seems like we have no major PRs on the docket.

(optionally, we could wait on #6726 or #6627; #6818 would make this a minor bump)

If we think this is a good idea, happy to perform the release (second time's the charm!).

**Comment by ybiquitous** [comment]
Sounds good! 👍🏼

**Comment by mattxwang** [comment]
I've published to npm; though, I don't have permissions to merge into the website (see: https://github.com/stylelint/stylelint.io/pull/318).

**Comment by mattxwang** [comment]
Confirmed that demo works as intended! Last item is the tweet (which I don't have access to) - @ybiquitous do you mind doing this when you have a chance? Then, we can close this issue!

**Comment by ybiquitous** [comment]
Sure. I'll tweet the new release soon. Just a minute. ✍🏼

**Comment by ybiquitous** [comment]
Tweeting is done. We completed all the tasks! 🎉 

Thank you for taking over the release task, @mattxwang. And thanks a lot for your contributions and efforts, everyone!
